ImputationPipeline/ScoringPart.wdl

```diff
@@ -254,10 +254,63 @@ task AdjustScores {
 		File array_scores
 		Int mem = 2
 	}
+	# Rscript ~{adjusting_Rscript} ~{population_pcs} ~{population_scores} ~{array_pcs} ~{array_scores}
+	command <<<
+		Rscript <<- "EOF"
+			library(ggplot2)
+			library(dplyr)
 
-	command {
-		Rscript ~{adjusting_Rscript} ~{population_pcs} ~{population_scores} ~{array_pcs} ~{array_scores}
-	}
+			args = commandArgs(trailingOnly=TRUE)
```

Comment:
I don't think this will work, since there aren't any args being passed.  But I think we should remove the use of command line args here, since we can just reference the task inputs directly in the R code, instead of passing them in through the command line.